dist/scripts/ctd-omega-editorial-frontend.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=CTD Omega Editorial Frontend
+Documentation=https://github.com/nationalarchives/ctd-omega-editorial-frontend
+After=syslog.target
+
+[Service]
+Type=simple
+User=ctd-web
+Group=ctd-web
+Environment="JAVA_HOME=/usr/lib/jvm/jre-1.8.0
+Environment="APPLICATION_SECRET=_mTFr@DIDmTUC9LFSFsn_j]iICDMj=DKMy/yuw/gy10RgsUiXuOK0beEP]SiCwMO"
```

Comment:
Are we leaving the secret in the repository until the application is deployed somewhere?
Maybe if/when this becomes an issue we could use `EnvironmentFile=` pointing to a file only readable by the ctd-web user.

Reply:
@felixgb That seems like a sensible approach. We should be able to use Terraform to create the file and put it on the server. I will check with @adamretter as I think he has done something like this previously.